### PR TITLE
refactor(depencencies): drop subversion and workaround

### PR DESCRIPTION
**Pull request by alxwr**
### PR progress checklist (to be filled in by reviewers)

- [ ] Changes to documentation are appropriate (or tick if not required)
- [ ] Changes to tests are appropriate (or tick if not required)
- [ ] Reviews completed

---

#### Primary type

- [ ] `[build]`    Changes related to the build system
- [ ] `[chore]`    Changes to the build process or auxiliary tools and libraries such as documentation generation
- [ ] `[ci]`       Changes to the continuous integration configuration
- [ ] `[feat]`     A new feature
- [ ] `[fix]`      A bug fix
- [ ] `[perf]`     A code change that improves performance
- [x] `[refactor]` A code change that neither fixes a bug nor adds a feature
- [ ] `[revert]`   A change used to revert a previous commit
- [ ] `[style]`    Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc.)

#### Secondary type

- [x] `[docs]`     Documentation changes
- [ ] `[test]`     Adding missing or correcting existing tests

### Does this PR introduce a `BREAKING CHANGE`?

No.

### Describe the changes you're proposing

This MR just removes an old, obsolete workaround.

### Debug log showing how the proposed changes work

States removed, so no output.

### Documentation checklist

- [ ] Updated the `README` (e.g. `Available states`).
- [ ] Updated `pillar.example`.

### Testing checklist

- [ ] Included in Kitchen (i.e. under `state_top`).
- [ ] Covered by new/existing tests (e.g. InSpec, Serverspec, etc.).
- [ ] Updated the relevant test pillar.

### CODE REVIEW

File: redmine/osfamilymap.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ Debian:
   web_user: www-data
   web_group: www-data
   dependencies:
-    - subversion
+    #- subversion
```
Comment (baby-gnu) [inline redmine/osfamilymap.yaml:5]: Maybe just remove the lines?

File: redmine/osfamilymap.yaml [hunk 2]
```diff
@@ -13,11 +13,11 @@ FreeBSD:
   web_user: www
   web_group: www
   dependencies:
-    - subversion
+    #- subversion
```
Comment (baby-gnu) [inline redmine/osfamilymap.yaml:16]: Like above, remove instead of comment?

File: pillar.example [hunk 3]
```diff
@@ -11,7 +11,7 @@ redmine:
     web_group: www-data
     # Packages which must be installed
     dependencies:
-      - subversion
+      #- subversion
```
Comment (baby-gnu) [inline pillar.example:14]: Maybe just remove the lines?

**Review by baby-gnu — APPROVED** [review]

File: redmine/osfamilymap.yaml (hunk 1 above)
Comment (alxwr) [reply redmine/osfamilymap.yaml:5]: Done. :-)

File: redmine/osfamilymap.yaml (hunk 2 above)
Comment (alxwr) [reply redmine/osfamilymap.yaml:16]: Done. :-)

File: pillar.example (hunk 3 above)
Comment (alxwr) [reply pillar.example:14]: Done. :-)

**Comment by alxwr** [comment]
Thanks for your review! :-)

**Pull request merged**